src/Migrations/Version20181106155324.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20181106155324 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->createTable('talk');
+        $table->addColumn('id', Type::GUID);
+        $table->addColumn('title', Type::STRING, ['length' => 255]);
+        $table->addColumn('description', Type::TEXT);
+        $table->addColumn('speaker_id', Type::GUID);
+    }
```

Comment:
I think you forget to define Primary Key on 'id' and Index on 'speaker_id'

Reply:
Oh you're right, thank you :)